readthedocs_ext/_static/javascript/client.js

```diff
@@ -1,11 +1,2 @@
-comm = require('./lib/comm')
-events = require('./lib/events')
-display = require('./lib/display')
-
 $(document).ready(function () {
-	events.initEvents();
-	comm.initOptions();
-	comm.initMetaData();
-	display.initDisplay();
 })
-
```

Comment:
All this file can be removed, I suppose. But the `lib/settings.js` was required by the others libs, so it was included in the bundle. Settings.js have information that is used outside the comments builder?

Reply:
Also, we need to re-minify the js (css too?) after the PR is completed.